extensions/library-genesis/metadata/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Library Genesis Changelog
```

Comment:
seems like a duplicated file from your already existing `CHANGELOG.md` in the project root, you can delete this one or move it to root and overwrite the other one